code/arith.py

```diff
@@ -6,28 +7,35 @@ def main():
 
     operator = sys.argv[1]
     assert operator in ['add', 'subtract', 'multiply', 'divide'], (
-        'Operator is not one of add, subtract, multiply, or divide: '
-        'bailing out')
+        '''Operator is not one of add, subtract, multiply, or divide:
+        bailing out''')
```

Comment:
I think the reason this was done with the adjacent single-quoted strings inside the parentheses is because with using the triple-quoted, you get that whitespace before "bailing out" as literal whitespace.

Reply:
Could you please elaborate on this one?